### PR TITLE
config: show all of installed JVM

### PR DESCRIPTION
**Pull request by xu-cheng**
Example output:
```
Java: 1.8.0_31, 1.6.0_65-b14-466.1
```

### CODE REVIEW

**Comment by mikemcquaid** [comment]
This doesn't match how we output other things e.g. Ruby. What's the benefit here; would this have made a particular use bug easier to solve?

File: Library/Homebrew/cmd/config.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'hardware'
+require 'rexml/document'
```
Comment (mikemcquaid) [inline Library/Homebrew/cmd/config.rb]: Yuck, XML parsing :sob:

**Comment by xu-cheng** [comment]
> This doesn't match how we output other things e.g. Ruby. What's the benefit here; would this have made a particular use bug easier to solve?

Not all the java depended formulae will link to the latest JVM. e.g. Java version sensitive formula . It can be (some) helpful to debug in this situation.

**Comment by mikemcquaid** [comment]
Have we seen any users with this problem?

**Comment by xu-cheng** [comment]
https://github.com/Homebrew/homebrew-science/issues/1899 may associate with it.

**Comment by mikemcquaid** [comment]
Hmm. I guess I'm probably :-1: on this unless having it in the output would have helped us solve a user problem easier.

**Pull request closed without merging**